src/algebra/big_operators.lean

```diff
@@ -773,6 +799,15 @@ begin
   exact add_lt_add_of_lt_of_le hlt (sum_le_sum $ λ j hj, Hle j  $ mem_of_mem_erase hj)
 end
 
+lemma sum_lt_sum_of_nonempty (hs : s.nonempty) (Hlt : ∀ x ∈ s, f x < g x) :
```

Comment:
Please provide a multiplicative version (and prove this one with `to_additive`).

Reply:
Hmmm, maybe the multiplicative version doesn't play well with the current setup of the algebraic hierarchy...

Reply:
Yeah I couldn't get this to work - I presume it's for the same reason that `sum_le_sum` etc are proved directly instead of using `to_additive`.

Reply:
Let's leave this for now.